src/actions/index.js

```diff
@@ -0,0 +1,18 @@
+import axios from "axios";
```

Comment:
Did you try to run this file with Prettier? The import is using double quotes instead single quote..

Reply:
Oh no, seems like I'm forcing it in my prettier, shame on me...